examples/tutorials/self-paced-training/part-1_federated_learning_introduction/chapter-2_develop_federated_learning_applications/02.3_convert_machine_learning_to_federated_learning/02.3.2_convert_kmeans_to_federated_learning/code/src/kmeans_trainer.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
2025
Personally preference suggest rename this file to "kmeans_fl.py"

Reply:
updated